commands/slash/invite.js

```diff
@@ -21,7 +21,7 @@ const command = new SlashCommand()
 								client.config.clientId
 							}&permissions=${
 								client.config.permissions
-							}&scope=${ client.config.scopes.toString().replace(/,/g, "%20") }`,
+							}&scope=bot%20applications.commands`
```

Comment:
Changing the scopes in config.js would do the same (in this case adding "bot" and removing identity and guilds if not required)
https://github.com/SudhanPlayz/Discord-MusicBot/blob/7041aa63265d2e3dad153946064c62600ef85e7d/config.js#L9

Plus that way it's more convenient if someone wants to change the scopes for some reason.

Reply:
in that case we need to add more scope such `inviteScope`, iirc the normal scope (`scope`) is in use with the dashboard so if we change the normal scope for invite link it may broke the dashboard